weaveworks/weave-gitops-enterprise
# Add new pipelines UI with promotion information

Closes #3312 
Closes #3313 
Closes #3324 

Figma link: https://www.figma.com/file/IVHnM9iyeFWpd11evtY8ux/Weave-GitOps?node-id=19656%3A13129&mode=dev

Faked Manual Pull Request Approval:
<img width="1254" alt="image" src="https://github.com/weaveworks/weave-gitops-enterprise/assets/65822698/a2c75185-3385-4dda-8e79-e0826bbe3f65">

Example Video:

https://github.com/weaveworks/weave-gitops-enterprise/assets/65822698/2faf315f-a1df-4970-b56f-4e3e024a65bc

How to test locally:
- `WEGO_EE_PROXY_HOST=https://pipelines.wge.dev.weave.works  yarn start` with demo01 cluster login

This PR gets the Pipelines UI about as close to the new designs as I can get it with pending backend changes on the way. 
Expected remaining work:
- flesh out promotion information box (needs error instructions supplied by backend)
- add promotion indicators in between environments with dashed lines
- add progressive delivery dropdown in targets with canaries
- change approve button icon
